frontend/src/views/Game.tsx

```diff
@@ -10,20 +16,45 @@ type LocationState = {
 function GamePage() {
   const location = useLocation<LocationState>();
   const [room, setRoom] = useState<Room | null>(null);
+  const [problems, setProblems] = useState<Problem[] | null>(null);
+  const [error, setError] = useState<string | null>(null);
 
   // Called every time location changes
   useEffect(() => {
     if (location && location.state && location.state.room) {
       setRoom(location.state.room);
     }
+    getProblems().then((res) => {
+      if (isError(res)) {
+        setError((res as ErrorResponse).message);
+        setProblems(null);
+      } else {
+        setProblems(res as Problem[]);
+        setError(null);
+      }
+    });
   }, [location]);
 
+  const firstProblem = problems?.[0];
+
   return (
     <div>
-      Room:
-      {' '}
-      {room ? room.roomId : 'No room joined'}
-      <Editor height="90vh" language="javascript" />
+      <Header />
+      <FlexContainer>
+        <FlexInfoBar>
+          Room:
+          {' '}
+          {room ? room.roomId : 'No room joined'}
+        </FlexInfoBar>
+        <FlexPanel>
+          <ProblemHeaderText>{ firstProblem?.name }</ProblemHeaderText>
+          <Text>{ firstProblem?.description }</Text>
+          <ErrorText>{error ? <ErrorMessage message={error} /> : null}</ErrorText>
```

Comment:
Got rid of the `<ErrorText>` tag; I didn't realize that the red style was a part of `<ErrorMessage>` already.

Reply:
Not sure if this has been updated already, but if not, this can simply be `{error ? <ErrorMessage message={error} /> : null}` (without the ErrorText wrapped around it).